### ISSUE TITLE
Errors in console when closing panel

### ISSUE THREAD

**Issue by seanzi86** [officert/vue-slideout-panel#47]
When no panel is displayed and you press the escape key, the following error is displayed in the console.

```
Uncaught TypeError: Cannot read property 'disableEscClick' of undefined
    at VueComponent.onEscapeKeypress
```
Looking at the source code, `onEscapeKeypress` , is always expecting currentPanel to return an object. If no panel is currently displayed then `this.panels[this.panels.length - 1]` will return null and this in turn causes the exception above.

I am happy to create a PR if you would like.

**Comment by officert** [comment]
Fixed in `2.4.0` : https://github.com/officert/vue-slideout-panel/releases/tag/2.4.0

**Comment by seanzi86** [comment]
Thank you for the prompt fix :)
